main.go

```diff
@@ -52,6 +57,11 @@ func main() {
 	}
 	log.Info("Application started")
 
+	// Validate input, as both of these being configured would be surprising.
+	if configDockerconfigjson == "" && configDockerConfigJSONPath == "" {
```

Comment:
should be `if configDockerconfigjson != "" && configDockerConfigJSONPath != ""`

Reply:
:+1: Yeah, definitely that's what it should have been. Thanks!

Reply:
Change was made in 3rd commit.